staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/plugin.go

```diff
@@ -69,7 +69,6 @@ func NewMutatingWebhook(configFile io.Reader) (*Plugin, error) {
 
 // SetScheme sets a serializer(NegotiatedSerializer) which is derived from the scheme
 func (a *Plugin) SetScheme(scheme *runtime.Scheme) {
-	a.Webhook.SetScheme(scheme)
 	if scheme != nil {
 		a.scheme = scheme
```

Comment:
what do we use this scheme for now? I'd expect it to have similar issues wherever it was used

Reply:
looks like we special-case unstructured in mutatingDispatcher#callAttrMutatingHook and avoid calling `a.scheme.New`... plumbing the request ObjectCreator there as well would likely be more correct